Libraries/LibWeb/WebVTT/VTTCue.h

```diff
@@ -42,6 +44,9 @@ class VTTCue final : public HTML::TextTrackCue {
     bool snap_to_lines() const { return m_snap_to_lines; }
     void set_snap_to_lines(bool snap_to_lines) { m_snap_to_lines = snap_to_lines; }
 
+    LineAndPositionSetting line() const { return m_line; }
+    void set_line(LineAndPositionSetting line) { m_line = line; }
```

Comment:
Similarly to `.position`, shouldn't we have a `computed_line()` as well?

Reply:
There [is](https://w3c.github.io/webvtt/#cue-computed-line), but so much of it would currently need to be FIXME'd.

Happy to add what could be done though :^) if you'd like

Reply:
Done :) To be fair, it's not as FIXME heavy as I seemed to think.